lib/common/options.c

```diff
@@ -229,6 +229,16 @@ static const pcmk__cluster_option_t cluster_options[] = {
     },
 
     // Fencing-related options
+    {
+        PCMK_OPT_FENCE_REMOTE_WITHOUT_QUORUM, NULL, PCMK_VALUE_BOOLEAN, NULL,
+        PCMK_VALUE_FALSE, pcmk__valid_boolean,
+        pcmk__opt_schedulerd|pcmk__opt_advanced,
+        N_("Whether remote nodes can be fenced without quorum"),
+        N_("By default, inquorate nodes can not fence Pacemaker Remote nodes "
```

Comment:
"...an inquorate node cannot fence Pacemaker Remote nodes that are part of its partition..."